docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/advanced-operations/monitoring.md

```diff
@@ -8,6 +8,12 @@ description: This guide explains about the IO engine pool metrics exporter.
 ---
 # Monitoring
 
+:::info
+A set of Grafana dashboards for Monitoring and visualization of exported metrics via various OpenEBS Storage are bundled with OpenEBS installation which is by default disabled.
+
+To enable the Grafana dashboards, see the instructions given in the [Installing OpenEBS documentation](../../../../quickstart-guide/installation.md#installation-via-helm).
```

Comment:
I also don't see the information about the list of available dashboards and their descriptions.

Reply:
This will be taken up later :)